### PR TITLE
feat: new styles for subtask in lab

### PR DESCRIPTION
**Pull request by kisielewskik**
## Description

Task from Alex Ronquillo from New Relic org. Purpose of this is making new styles for lab procedures while we have parrarel sub  procedures.

## Reviewer Notes

For testing it you have to make procedure with not-integer ProdIdx, example: ProcIdx 2.1, should be under ProcIdx 2 with simpler styles.

### CODE REVIEW

**Review by DominikMarciniszyn — DISMISSED** [review]

File: src/templates/LabOverviewTemplate.js [hunk 1]
```diff
@@ -47,21 +47,29 @@ const LabOverviewTemplate = ({ data, location }) => {
...
-                      description={
-                        frontmatter.tileShorthand?.description ||
-                        frontmatter.description
-                      }
-                    />
-                  ))}
+                  .map(({ fields, frontmatter }, index) => {
+                    const procIdxisNotInteger = Number.isInteger(
+                      frontmatter.procIdx
+                    );
```
Comment (LizBaker) [inline src/templates/LabOverviewTemplate.js:53]: as this is set up now procIdxisNotInteger will evaluate to true when a number is an integer. would you mind renaming it to procIdxIsInteger?

Comment (kisielewskik) [reply src/templates/LabOverviewTemplate.js]: I changed name of variable is more logic now

File: src/components/GuideTile/GuideTile.js [hunk 2]
```diff
@@ -13,6 +20,8 @@ const GuideTile = ({ duration, title, description, className, to }) => (
     css={css`
       display: grid;
       grid-template-rows: auto 1fr auto;
+      width: ${procIdxIsInteger ? '100%' : '90%'};
+      margin-left: ${procIdxIsInteger ? '0' : '10%'};
```
Comment (LizBaker) [inline src/components/GuideTile/GuideTile.js:24]: changing the styles here have had some unintended consequences in other places that we use the GuideTiles. I think moving these styles to the LabOverviewTemplate might be best. you can use the css prop within that guidetile just like here and select for the p and h3

normal styling:
Screen Shot 2022-02-04 at 9 15 26 AM

current changes:
Screen Shot 2022-02-04 at 9 15 34 AM

Comment (jpvajda) [reply src/components/GuideTile/GuideTile.js:24]: That spacing between the tiles seems really wide I'd recommend following the pattern we use for space between the quickstart tiles on I/O 

https://developer.newrelic.com/instant-observability/
Screen Shot 2022-02-04 at 11 00 32 AM

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: src/components/GuideTile/GuideTile.js (hunk 2 above)
Comment (kisielewskik) [reply src/components/GuideTile/GuideTile.js:24]: Totalny agree, I didn’t noticed that change will have so big impact on other views, i will move that styles to LabOverviewTemplate asap to keep website pattern, thank you for your advices

File: src/templates/LabOverviewTemplate.js [hunk 3]
```diff
@@ -47,21 +47,29 @@ const LabOverviewTemplate = ({ data, location }) => {
...
-                      title={`${frontmatter.procIdx}. ${
-                        frontmatter.tileShorthand?.title || frontmatter.title
-                      }`}
-                      description={
-                        frontmatter.tileShorthand?.description ||
-                        frontmatter.description
-                      }
-                    />
-                  ))}
+                  .map(({ fields, frontmatter }, index) => {
```
Comment (jpvajda) [inline src/templates/LabOverviewTemplate.js:50]: Attached is the screen shot for this change so it's visible, this is OK for me, so I approve this.
Screen Shot 2022-02-04 at 9 47 45 AM

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: src/components/GuideTile/GuideTile.js (hunk 2 above)
Comment (jpvajda) [reply src/components/GuideTile/GuideTile.js:24]: Thanks @kisielewskik!

**Review by LizBaker — APPROVED** [review]

**Pull request merged**